app/src/main/kotlin/cz/covid19cz/erouska/net/ExposureServerRepository.kt

```diff
@@ -37,7 +37,7 @@ class ExposureServerRepository @Inject constructor(
 
     companion object {
         private val KEY_EXPORT_INDEX = if (BuildConfig.FLAVOR == "dev") {
-            "${AppConfig.keyExportUrl}/index.txt"
+            "${AppConfig.keyExportUrl}index.txt"
```

Comment:
No, this is intentional. The url needs double slashes - try it yourself, it doesn't work without them.

Reply:
ok, perhaps it was okay that the file was not found, then, closing.